### PR TITLE
Implemented selection sort algorithm (#6)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5,12 +5,14 @@
 "@babel/code-frame@^7.0.0":
```
Comment (euclid1990) [inline yarn.lock]: Please uncommit this file 🙇

File: sorting/selection-sort.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ * @example
+ * console.log(selectionSort([2, 5, 1, 0, 4])); // [ 0, 1, 2, 4, 5 ]
+ *
+ * @public
+ * @module sorting/bubblesort
+ * @param {Array} array Input array.
+ * @return {Array} Sorted array.
+ */
+function selectionSort(array) {
+  var minIdx;
```
Comment (euclid1990) [inline sorting/selection-sort.js]: I think we should use `let` to declare variable than using `var` keyword

File: sorting/selection-sort.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  var temp;
+  var len = array.length;
+  for (var i = 0; i < len; i++) {
+    minIdx = i;
+    for (var j = i + 1; j < len; j++) {
+      if (array[j] < array[minIdx]) {
+        minIdx = j;
+      }
+    }
+    temp = array[i];
```
Comment (euclid1990) [inline sorting/selection-sort.js]: please separate to swap function !

File: yarn.lock (hunk 1 above)
Comment (ghost) [reply yarn.lock]: Oh, my bad, sorry!

File: sorting/selection-sort.js (hunk 2 above)
Comment (ghost) [reply sorting/selection-sort.js]: Fixed!

File: yarn.lock (hunk 1 above)
Comment (euclid1990) [reply yarn.lock]: @TheNullable Uncommit is not remove this file, please use `git checkout -- yarn.lock`

**Pull request closed without merging**